pkg/configtx/example_test.go

```diff
@@ -564,6 +564,29 @@ func ExampleConfigTx_SetConsortiumMSP() {
 	}
 }
 
+// This example shows the addition of a new consortium.
+func ExampleConfigTx_AddConsortium() {
+	baseConfig := fetchChannelConfig()
+	c := configtx.New(baseConfig)
+
+	org, err := c.ApplicationOrg("Org1")
+	if err != nil {
+		panic(err)
+	}
+
+	newConsortium := configtx.Consortium{
+		Name: "SampleConsortium2",
+		Organizations: []configtx.Organization{
+			org,
```

Comment:
It may be improper as an example here to add an application org as the consortium org for the example. Let's create a new org for the sole purpose of adding it to the consortium.